### PR TITLE
git-vendor: init at 1.2.0

### PR DESCRIPTION
**Pull request by Profpatsch**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Added a release notes entry if the change is major or breaking
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/version-management/git-and-tools/git-vendor/default.nix [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  '';
+
+  meta = {
+    description = "A git command for managing vendored dependencies";
+    longDescription = ''
+      git-vendor is a wrapper around git-subtree commands for checking out and updating vendored dependencies.
+
+      By default git-vendor conforms to the pattern used for vendoring golang dependencies:
+
+          Dependencies are stored under vendor/ directory in the repo.
```
Comment (sternenseemann) [inline pkgs/applications/version-management/git-and-tools/git-vendor/default.nix:57]: The list bullets didn't quite make it.

**Review by sternenseemann — APPROVED** [review]
LGTM other than the longDescription formatting mishap.

File: pkgs/applications/version-management/git-and-tools/git-vendor/default.nix [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+in stdenv.mkDerivation {
+  pname = "git-vendor";
+  inherit version;
+
+  src = fetchFromGitHub {
+    owner = "brettlangdon";
+    repo = "git-vendor";
+    rev = "v${version}";
+    inherit sha256;
```
Comment (SuperSandro2000) [inline pkgs/applications/version-management/git-and-tools/git-vendor/default.nix:16]:
```suggestion
stdenv.mkDerivation rec {
  pname = "git-vendor";
  version = "1.2.0";

  src = fetchFromGitHub {
    owner = "brettlangdon";
    repo = "git-vendor";
    rev = "v${version}";
    sha256 = "1z9fmrfxqi56pj7f1506q2z41crz702jk88gv57baf6fz63m93v2";
```

File: pkgs/applications/version-management/git-and-tools/git-vendor/default.nix [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+          Dependencies are stored under the fully qualified project path.
+              e.g. https://github.com/brettlangdon/forge will be stored under vendor/github.com/brettlangdon/forge.
+    '';
+    homepage = "https://github.com/brettlangdon/git-vendor";
+    license = lib.licenses.mit;
+    maintainers = [ lib.maintainers.Profpatsch ];
+    platforms = lib.platforms.all;
+  };
+
+}
```
Comment (SuperSandro2000) [inline pkgs/applications/version-management/git-and-tools/git-vendor/default.nix:67]:
```suggestion
  };
}
```

File: pkgs/applications/version-management/git-and-tools/git-vendor/default.nix (hunk 1 above)
Comment (Profpatsch) [reply pkgs/applications/version-management/git-and-tools/git-vendor/default.nix:57]: thx

File: pkgs/applications/version-management/git-and-tools/git-vendor/default.nix (hunk 2 above)
Comment (Profpatsch) [reply pkgs/applications/version-management/git-and-tools/git-vendor/default.nix:16]: Can you please stop that? It’s annoying to no end.

**Pull request merged**